### PR TITLE
feat!: Change CGU to CGS

### PR DESCRIPTION
**Pull request by taorepoara**
## About this PR
Rename CGU (terms of use) to CGS (terms of service) in the project and add the latest version of the terms of service.

## Technical highlight/advice
The rename in database needed some SQL request.

## How to test my changes
Just start the server and test it.

## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I did not include breaking changes
- [x] I made my own code-review before requesting one

### I included unit tests that cover my changes
- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### I added/updated the documentation about my changes
- [ ] 📜 README.md
- [x] 📕 docs/*.md
- [ ] 📓 docs.lenra.io
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: docs/cgs-management.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+The terms and conditions of use (CGS in french) are managed thanks to markdown files.
```
Comment (jonas-martinez) [inline docs/cgs-management.md:1]:
```suggestion
The terms and conditions of service (CGS in french) are managed thanks to markdown files.
```

File: apps/lenra/priv/repo/migrations/20220307160000_user_accept_cgu_with_constraint.exs [hunk 2]
```diff
@@ -11,7 +11,7 @@ defmodule Lenra.Repo.Migrations.UserAcceptCguWithConstraint do
         IF last_version_id = NEW.cgu_id THEN
           RETURN NEW;
         ELSE
-          RAISE EXCEPTION 'Not latest CGU';
+          RAISE EXCEPTION 'Not latest CGS';
```
[the remark this answers is not in the document]
Comment (taorepoara) [reply apps/lenra/priv/repo/migrations/20220307160000_user_accept_cgu_with_constraint.exs:14]: My bad, not seen this one.

[the remark this answers is not in the document]
Comment (taorepoara) [reply apps/lenra/priv/repo/migrations/20220307160000_user_accept_cgu_with_constraint.exs:14]:
```suggestion
          RAISE EXCEPTION 'Not latest CGU';
```

**Review by jonas-martinez — APPROVED** [review]

**Pull request merged**